core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -222,7 +234,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("consumer_prop")
       .ofType(classOf[String])
-    val consumerConfigOpt = parser.accepts("consumer.config", s"Consumer config properties file. Note that ${consumerPropertyOpt} takes precedence over this config.")
+    val consumerConfigOpt = parser.accepts("consumer.config", "Consumer config properties file.")
```

Comment:
Why did the warning about the override get removed? That seems important to have since its hard to know precedence unless you look at the code.